xclim/sdba/adjustment.py

```diff
@@ -483,23 +483,28 @@ def __init__(
         cluster_thresh: Quantity (str with units)
           The threshold value for defining clusters.
         q_thresh : float
-          The quantile of "extreme" values.
+          The quantile of "extreme" values, [0, 1[.
         dist : str
           Name of the distribution (scipy name) to use.
 
         In adjustment:
 
-        frac: float, [0, 1]
+        frac: float
           Fraction where the cutoff happens between the original scen and the corrected one.
-          See Notes.
+          See Notes, ]0, 1].
         power: float
           Shape of the correction strength, see Notes.
 
         Notes
         -----
-
-
-        Correction strength:
+        Extreme values are extracted from `ref`, `hist` and `sim` by finding all "clusters",
+        i.e. runs of consecutive values above `cluster_thresh`. The `q_thresh`th percentile
+        of these values is taken on `ref` and `hist` and becomes `thresh`, the extreme value
+        threshold. The maximal value of each cluster of `ref`, if it exceeds that new threshold,
+        is taken and a `dist` (default : Generalized Pareto) distribution is fitted to them.
```

Comment:
Here, if you take the exceedence approach, the only proper default is a Generalized Pareto. So, if you change the default, the approach should not be an exceedence approach.

Reply:
I suggest that, for now, we force the pareto and disable the option to change the distribution. Opening to other distributions would require more testing, changes to the _clustering_ algorithm and possibly other more subtle changes.